### ISSUE TITLE
Left part of focus ring covers up the points tally in the app bar

### ISSUE THREAD

**Issue by sairina** [learningequality/kolibri#9180]
## Observed behavior
The focus ring around the user profile/options button covers up part of the points.

<img width="1104" alt="Screen Shot 2022-03-11 at 1 57 28 PM" src="https://user-images.githubusercontent.com/13563002/158239375-974279af-ff4f-4833-bd93-22093a1f2385.png">

## Errors and logs
None

## Expected behavior
There should be space between the focus ring and the points tally.

## User-facing consequences
It's more a stylistic issue.

## Steps to reproduce
1. Login as learner
2. Use the keyboard to navigate through the app bar
3. Notice that once you get to the user profile icon, the focus covers the points

## Context
Kolibri 0.15
MacOS
Chrome

**Comment by marcellamaki** [comment]
With the app bar changes and elimination of this dropdown menu, this should be no longer relevant with those updates. Adding to this epic just to track. No separate updates should be required, and this issue can be closed following the app bar updates being merged.

**Comment by rtibbles** [comment]
This is no longer extant for the reasons noted by @marcellamaki above.
